doc/user/command-line.xml

```diff
@@ -678,6 +679,46 @@ foo.in
         <scons_output_command>scons -Q -n --prefix=/tmp/install</scons_output_command>
       </scons_output>
 
+      <note>
+      <para>
+      Long options and option-arguments separated by whitespace,
+      rather than by <literal>=</literal>, are ambiguous.
```

Comment:
Instead of "Ambiguous" can you change to something like "is not possible for SCons to determine if this is a requested target, or the value for the argument".  Since (to me) in this context ambiguous is ambiguous.. ;)

Reply:
well, ambiguous means the meaning cannot be determined :-)  "open to more than one interpretation".  I can certainly reword.